Update README (#31)
f-klubben/sangbog: README.md

# Sangbogen

> This is the current songbook of F-klubben.

Reviewer: ```suggestion
This is the current songbook in the F-Club.
```

Reviewer: F-club*

Reviewer: I give up

## Download

The latest songbook can be downloaded from [here](https://github.com/f-klubben/sangbog/releases/latest/download/sangbog.pdf) in booklet format ready for printing. If you require the songbook in a non-booklet format, you can follow the build instructions below (and use the `pdf` target for`make`).

## Building from source

For Debian-based systems (in other distributions package names may vary):

1. Fetch the source code

```sh
git clone https://github.com/f-klubben/sangbog.git
```

2. Install prerequisites:

```sh
sudo apt install texlive-latex-extra psutils texlive-lang-european ghostscript
```

3. Build the sangbog
   `make booklet` for booklet format, `make pdf` for non-booklet (continuous) format

## Adding new songs

> =======

Reviewer: ![billede](https://github.com/f-klubben/sangbog/assets/17174333/bc44bfa2-5f77-4a84-bc47-c0af8f18cc86)
I'm not sure how this is supposed to look

Author: Ooops, Imma fix it.
